python-sdk/tutorials/nuscenes_lidarseg_tutorial.ipynb

```diff
@@ -21,19 +21,43 @@
     "- ```lidarseg_category.json```\n",
     "- ```lidarseg.json```\n",
     "\n",
-    "Copy the ```lidarseg``` folder (this contains .bin files; each .bin file is an annotation for a pointcloud) into ```/data/sets/nuscenes/```, and copy ```lidarseg_category.json``` and ```lidarseg.json``` into ```/data/sets/nuscenes/v1.0-mini```.\n",
+    "Copy the `lidarseg` folder (this contains `.bin` files; each `.bin` file is an annotation for a pointcloud) into `/data/sets/nuscenes/`, and copy `lidarseg_category.json` and `lidarseg.json` into `/data/sets/nuscenes/v1.0-mini`.\n",
```

Comment:
Let's remove the trailing slash from `/data/sets/nuscenes/`.

Reply:
Remove `and copy `lidarseg_category.json` and `lidarseg.json` into `/data/sets/nuscenes/v1.0-mini`` and provide the .zip file with those subfolders.